packages/runtime-core/__tests__/components/Teleport.spec.ts

```diff
@@ -299,4 +302,55 @@ describe('renderer: teleport', () => {
     )
     expect(serializeInner(target)).toBe('')
   })
+
+  test('should work with block tree', async () => {
+    const target = nodeOps.createElement('div')
+    const root = nodeOps.createElement('div')
+    const disabled = ref(false)
+
+    const App = {
+      render() {
+        return (
+          openBlock(),
+          createBlock(
+            Fragment,
+            null,
+            [
+              h(
+                Teleport,
+                { to: target, disabled: disabled.value },
+                h('div', 'teleported')
+              ),
+              h('div', 'root')
+            ],
+            PatchFlags.STABLE_FRAGMENT
+          )
+        )
+      }
+    }
+    render(h(App), root)
+    expect(serializeInner(root)).toMatchInlineSnapshot(
+      `"<!--teleport start--><!--teleport end--><div>root</div>"`
+    )
+    expect(serializeInner(target)).toMatchInlineSnapshot(
+      `"<div>teleported</div>"`
+    )
+
+    disabled.value = true
+    await nextTick()
+    expect(serializeInner(root)).toMatchInlineSnapshot(
+      `"<!--teleport start--><div>teleported</div><!--teleport end--><div>root</div>"`
```

Comment:
This was fail in test, but I don‘t know why.The 250 line invoked, but the dom tree not update and not invoke the 260 code(and not throw errors in debug). 
https://github.com/vuejs/vue-next/blob/6efb2fe3ea2586c2dae065c09de7039c2d46c509/packages/runtime-core/src/components/Teleport.ts#L248-L262